### PR TITLE
Language edits and bugfixes

### LINKED ISSUE #171: An option to overwrite existing file with costools.timefilter.TimelineFilter
Hi, 

May I suggest adding a bool-type parameter "clobber" to costools.timefilter.TimelineFilter so that the output file can be overwritten. Currently, if I repeat the analysis without deleting the file created by previous runs, I will encounter a RuntimeError. 

Junjie

### PR DESCRIPTION
**Pull request by nkerman**
This PR contains the first batch of major changes since the repo was published - conforming to 2 categories: (**_1_**) **language edits intended to formalize or clarify the text** and (**_2_**) **bug fixes to users' issues**.
1. The Notebooks were originally written to be a bit more conversational than formal, and this lead to some areas with reduced clarity. As such, a batch of edits were made to eliminate figures of speech, idioms, etc.
2. We address/ Fix #171 and #173 (both on DayNight.ipynb)
* #171  focuses on overwrite functionality
* #173 ensures that the right type of data (`TIME-TAG`) is selected by the user before running a function

### CODE REVIEW

**Review by nkerman — COMMENTED** [review]
Small, looks good, no comments necessary.

**Review by nkerman — COMMENTED** [review]
No review necessary

**Review by nkerman — COMMENTED** [review]
c3eb359No review necessary

**Review by nkerman — COMMENTED** [review]
1216ad3 Bethan examined the language changes and approved, additionally giving me approval to make similar changes to the other released Notebooks

**Review by nkerman — COMMENTED** [review]
Minor text edits - no review necessary

**Review by nkerman — COMMENTED** [review]
Text changes - no review necessary

**Review by nkerman — COMMENTED** [review]
24d3d56 Text changes - no review necessary

**Review by nkerman — COMMENTED** [review]
0ede4a8 is relevant as the predecessor to 094c77f

**Review by nkerman — COMMENTED** [review]
@larrybradley @eteq 
Please see this commit 094c77f: which fixes issue #171 by checking for and deleting previously-created files. Supersedes commit 0ede4a8 which created the basic code for this change, but kept it as markdown.

The commit also changes a couple details in several examples (reversing an inequality to make more relevant to a user).

**Comment by nkerman** [comment]
@larrybradley @eteq
Please also see commit 5ca93ef: which fixes issue #173 by checking that all the files are taken in `TIME-TAG` mode.

The commit also fixes a couple typos/language changes.

File: notebooks/COS/DayNight/DayNight.ipynb [hunk 1]
```diff
@@ -208,32 +313,46 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "execution": {
```
Comment (eteq) [inline notebooks/COS/DayNight/DayNight.ipynb:317]: I noticed a lot of these `"exection"` entries here.  I'm not sure what that means, but is it possible you didn't clear outputs or something on one of these notebooks? (if you're not sure either, it's probably not an issue - just thought I'd check)

**Review by eteq — APPROVED** [review]
Looks fine to me - a couple minor comments you can address or not as you judge fit, @nkerman !

File: notebooks/COS/DayNight/DayNight.ipynb [hunk 2]
```diff
@@ -170,24 +226,73 @@
...
+     "iopub.status.busy": "2021-08-13T15:07:24.355833Z",
+     "iopub.status.idle": "2021-08-13T15:07:24.840662Z",
+     "shell.execute_reply": "2021-08-13T15:07:24.841182Z"
+    }
+   },
    "outputs": [],
    "source": [
+    "# If the files already exist, delete them so you may run the filter again:\n",
+    "if os.path.exists(intermediate_dir / \"filtered_corrtag_a.fits\"): # Check if you have created a filtered FUVA file\n",
+    "    os.remove(intermediate_dir / \"filtered_corrtag_a.fits\")      #   if so, delete the file so we can re-create it with TimelineFilter\n",
```
[the remark this answers is not in the document]
Comment (nkerman) [reply notebooks/COS/DayNight/DayNight.ipynb:288]: I'll look into this!

File: notebooks/COS/DayNight/DayNight.ipynb (hunk 1 above)
Comment (nkerman) [reply notebooks/COS/DayNight/DayNight.ipynb:317]: Ah, I believe this is from the way that I cleared my cells - I wrote a script that clears Notebooks with `jupyter-nbconvert --ClearOutputPreprocessor.enabled=True --inplace <Notebook.ipynb>` which I believe leaves them like that.

File: notebooks/COS/DayNight/DayNight.ipynb (hunk 2 above)
[the remark this answers is not in the document]
Comment (nkerman) [reply notebooks/COS/DayNight/DayNight.ipynb]: I've made these changes. Thanks for the suggestion!

**Comment by nkerman** [comment]
I've updated based on feedback from reviewer @eteq, now merging the PR.

**Pull request merged**